### PR TITLE
Add `ChimeNotifier`

### PR DESCRIPTION
**Pull request by cjames23**
Feature: 
Adding `ChimeNotifier` for Amazon provider to enable notifications to a Chime chat room via callbacks. This feature will allow a user to create notifications via a Chime webhook utlizing the `ChimeWebhookHook` with the message allowing templating. 

Example code added to How-to Guide for Chime notifications under the Amazon provider docs under notifications folder.

### CODE REVIEW

File: docs/apache-airflow-providers-amazon/notifications/chime_notifier_howto_guide.rst
Comment (eladkal) [inline docs/apache-airflow-providers-amazon/notifications/chime_notifier_howto_guide.rst:1]: The guide needs to be linked to the index
see example
https://github.com/apache/airflow/pull/32006

Comment (cjames23) [reply docs/apache-airflow-providers-amazon/notifications/chime_notifier_howto_guide.rst:1]: Fixed in the latest commit.

File: airflow/providers/amazon/aws/notifications/chime.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```
Comment (vincbeck) [inline airflow/providers/amazon/aws/notifications/chime.py:1]: Should we name the directory `notifiers` instead? To be consistent with `operators`, `hooks`, ...

Comment (cjames23) [reply airflow/providers/amazon/aws/notifications/chime.py:1]: Notifications was consistent with how other providers have named such as Slack and Discord. But I am open to renaming if there is consensus.

Comment (eladkal) [reply airflow/providers/amazon/aws/notifications/chime.py:1]: Lets stay with the current convention for now.
Notifiers is a new experience and I prefer to gather feedback before we start deprecating.

Comment (vincbeck) [reply airflow/providers/amazon/aws/notifications/chime.py:1]: Sounds good

**Review by vincbeck — APPROVED** [review]

**Review by eladkal — APPROVED** [review]

**Pull request merged**